src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
+        Machine [] machines = new Machine[] {bulldozer,excavator,truck};
```

Comment:
Please ensure proper spacing after commas in the array initialization to adhere to Java code style guidelines. It should be `new Machine[] {bulldozer, excavator, truck};`